src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2314,8 +2314,8 @@ private function addMailerSection(ArrayNodeDefinition $rootNode, callable $enabl
                             ->canBeEnabled()
                             ->info('S/MIME encrypter configuration')
                             ->children()
-                                ->scalarNode('certificate')
-                                    ->info('Path to certificate (in PEM format without the `file://` prefix)')
+                                ->scalarNode('repository')
```

Comment:
For reviewers, this change is fine as this feature has been introduced in 7.3, so not yet released